test/testdata/generate.sh

```diff
@@ -113,7 +113,7 @@ function scaffold_test_project {
       $kb create api --group example.com --version v1alpha1 --kind Busybox --image=busybox:1.28 --plugins="deploy-image/v1-alpha" --make=false
```

Comment:
Hi @jason1028kr,

That is great! thank you for your contribution.

It is just missing one detail. Could you please also update the name in the tests: https://github.com/kubernetes-sigs/kubebuilder/blob/d887e25e0a62eae1fd5882267bc3a53535198c6d/test/testdata/test.sh#L47